.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 /server/server.exe
 /server/server
+CHANGELOG.md
```

Comment:
Why do we need to ignore CHANGELOG.md? There is no such file and if we add it should not be ignored

Reply:
I use [this tool](https://github.com/skywinder/github-changelog-generator) to generate the release notes. It creates/updates a CHANGELOG.md file every time it's run, and I just want to ignore it so I don't have to delete it/think about it if I run it more than once.